components/provisioning/warewulf/SPECS/warewulf.spec

```diff
@@ -244,16 +229,52 @@ getent group %{wwgroup} >/dev/null || groupadd -r %{wwgroup}
 %dir %{_docdir}/warewulf
 %license %{_docdir}/warewulf/LICENSE.md
 
-%if %{api}
-%{_bindir}/wwapi*
-%config(noreplace) %{_sysconfdir}/warewulf/wwapi*.conf
-%endif
-
 ## Inserted by OHPC
 %if 0%{?rhel}
 %{tftpdir}
 %endif
+## End OHPC
+
+%package dracut
+Summary: dracut module for loading a Warewulf image
+BuildArch: noarch
+
+Requires: dracut
+%if 0%{?suse_version}
+%else
+Requires: dracut-network
+%endif
+Requires: curl
+Requires: cpio
+Requires: dmidecode
+
+%description dracut
+Warewulf is a stateless and diskless provisioning system for large clusters of
```

Comment:
Is this description needed here ?
I think only the `dracut` related text should be here.

Reply:
This is pulled directly from the original SPEC file.  Do you think we should remove it in the OpenHPC version?

Reply:
I see value in staying close to the upstream SPEC file. Makes changes in the future easier.